src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  border-radius: 4px;
+  margin: 0;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+}
+
+.nav__item {
+  display: flex;
+  justify-content: space-around;
+  align-items: stretch;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: Roboto, sans-serif;
```

Comment:
move it to the html selector

Reply:
I didn't manage such changes because they destroy current position of elements and it was indicated in Readme "Don't set height for `header` explicitly. Let the content (links) dictate it."
ff was moved to body